### PR TITLE
cli: more loading refactoring

### PR DESCRIPTION
**Pull request by krzysztof**
* Adds option for loading just a single record (by recid) from the dumps.

Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

File: invenio_migrator/cli.py [hunk 1]
```diff
@@ -117,38 +144,58 @@ def inspectrecords(sources, recid, entity=None):
...
 
 
-def loadcommon(sources, load_task, asynchronous=True, task_args=None,
-               task_kwargs=None):
+def loadcommon(sources, load_task, asynchronous=True, predicate=None,
+               task_args=None, task_kwargs=None):
     """Common helper function for load simple objects.
 
-    Note: Extra `args` and `kwargs` are passed to the `load_task` function.
+    NOTE: `task_args` and `task_kwargs` are passed to the `load_task`
```
Comment (hachreak) [inline invenio_migrator/cli.py]: could you use `.. note::` instead of `NOTE:`? :)
See also line 154.

Comment (krzysztof) [inline invenio_migrator/cli.py]: Nessun problema!

File: invenio_migrator/cli.py [hunk 2]
```diff
@@ -41,26 +41,53 @@ def dumps():
     """Migration commands."""
 
 
+def _loadrecord(record_dump, source_type, eager=False):
+    """Load a single record into the database.
+
+    :param record_dump: Record dump.
+    :type record_dump: dict
+    :param source_type: 'json' or 'marcxml'
+    :param eager: If True execute the task synchronously.
```
Comment (hachreak) [inline invenio_migrator/cli.py]: Could you add the quotes? 
Like 
```
``True``
```

**Comment by lnielsen** [comment]
Can you rebase?

**Comment by krzysztof** [comment]
@lnielsen It's rebased, the "cli refactoring" commit is a different refactoring, I intended to have two in previous commit but somehow it got merged before.

**Comment by lnielsen** [comment]
Branch is still out of date.

**Comment by krzysztof** [comment]
Spooky stuff happened with this PR

**Pull request closed without merging**